WoW-Realm-Tracker/Models/RealmsRequest.swift

```diff
@@ -0,0 +1,19 @@
+import Result
+import Argo
+import Swish
+
+struct RealmsRequest: Request {
+    typealias ResponseType = [Realm]
+
+    let requestUrl = "http://us.battle.net/api/wow/realm/status"
+
+    func build() -> NSURLRequest {
+        let url = NSURL(string: requestUrl)!
+        return NSURLRequest(URL: url)
+    }
+
+    func parse(j: JSON) -> Result<[Realm], NSError> {
+        let realms: Decoded<[Realm]> = j <|| "realms"
+        return .fromDecoded(realms)
```

Comment:
I think you can put the `<||` inside the parens here, like `.fromDecoded(j <|| "realms")`, what do you think?